internal/api/external.go

```diff
@@ -380,17 +384,17 @@ func (a *API) createAccountFromExternalIdentity(tx *storage.Connection, r *http.
 				externalURL := getExternalHost(ctx)
 				if terr = sendConfirmation(tx, user, mailer, config.SMTP.MaxFrequency, referrer, externalURL, config.Mailer.OtpLength, models.ImplicitFlow); terr != nil {
 					if errors.Is(terr, MaxFrequencyLimitError) {
-						return nil, tooManyRequestsError("For security purposes, you can only request this once every minute")
+						return nil, tooManyRequestsError(ErrorCodeOverEmailSendRateLimit, "For security purposes, you can only request this once every minute")
```

Comment:
/nit not urgent but can consider swapping this out with a string in the future. In  `recover.go`,  `user.go`  and `magic_link.go` it looks like we use `60 seconds` instead of `minute`